contracts/finality/src/state/finality.rs

```diff
@@ -110,6 +110,57 @@ pub fn insert_finality_sig_and_signatory(
     Ok(())
 }
 
+const MAX_PRUNING: u32 = 30;
+const DEFAULT_PRUNING: u32 = 10;
```

Comment:
Do you think we could be more aggressive here, given that the use case of this handler is to prune data up until a certain height